packages/electron-app/package.json

```diff
@@ -37,7 +42,11 @@
     "cross-env": "^5.2.0",
     "electron": "^4.0.0",
     "electron-builder": "^20.38.4",
+    "electron-devtools-installer": "^2.2.4",
     "electron-webpack": "^2.6.1",
     "webpack": "^4.28.3"
+  },
+  "dependencies": {
+    "@types/electron-devtools-installer": "^2.2.0"
```

Comment:
I think since electron-devtools-installer is a devDep, the types could also be devDeps